iroha_schema/src/lib.rs

```diff
@@ -36,6 +37,12 @@ pub trait IntoSchema {
     fn schema(metamap: &mut MetaMap);
 }
 
+/// Applicable for types that represents scale of fixed point
+pub trait ScaleAware {
+    /// scale of fixed point
+    fn scale() -> usize;
+}
```

Comment:
Maybe let's change the name not to confuse with SCALE codec?

Reply:
ok